src/config/settings.js

```diff
@@ -10,7 +10,10 @@ export const ENABLE_MOCK_API =
   process.env.REACT_APP_ENABLE_MOCK_API === 'true' ?? false;
 
 export const MODELS_ENDPOINT = 'https://api.sketchfab.com/v3/search';
```

Comment:
I would rename this to `MODEL_SEARCH_ENDPOINT`, and the new one to `MODEL_INFO_ENDPOINT` otherwise the confusion is too big with only the singular and plural forms.